SEED-platform/seed
# Major Workflow Improvements

#### What's this PR do?
- No more manual SCSS compilation or committing generated files (now handled automatically by Django)
- Bower removed and replaced with npm
- Remote fonts and scripts are now all local
- Landing files are now all in the same location
- Gulp workflow simplified to linting
- Lint fixes
- Removed unused files
- Fixed bugs
- Updated landing background
#### How should this be manually tested?
- Run `./bin/install_javascript_dependencies.sh`
- Monkey test, check that everything still works as expected (dev & prod settings)
